facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -153,16 +154,18 @@ public static void main(String[] args) {
                 engine.run(new StateHeadlessSetup());
             } else if (loadLastGame) {
                 engine.initialize(); //initialize the managers first
-                engine.getFromEngineContext(ThreadManager.class).submitTask("loadGame", () -> {
+
+                Mono.fromRunnable(() -> {
                     GameManifest gameManifest = getLatestGameManifest();
                     if (gameManifest != null) {
                         engine.changeState(new StateLoading(gameManifest, NetworkMode.NONE));
                     }
-                });
+                }).subscribeOn(GameScheduler.boundedElastic()).subscribe();
```

Comment:
I'm curious why this isn't just `GameScheduler.boundedElastic().schedule(() -> { ... })` - aren't they equivalent?

Reply:
this should let us do some tractability if we enable it.

Reply:
How does that work?

Reply:
https://projectreactor.io/docs/core/release/reference/#_tags

Reply:
I don't get it. That link talks about how we can add extra tags, but I don't see any of that here?